### PR TITLE
Improve doc and terminology

### LINKED ISSUE #2471: Terminology unification: suite server program
Ref https://github.com/cylc/cylc/pull/2469#issuecomment-341659520 and preceding discussion.

Throughout the documentation a running suite server program is variously referred to as "cylc", "the suite", "the suite daemon",  "the suite server", and "the suite server process" (and maybe more). 

These are all more or less inaccurate or ambiguous terms. They should all be replaced with "**the suite server program**" (which is both accurate and unambiguous, if slightly unwieldy).

### LINKED ISSUE #2499: Don't print pgrep advice if options not supported locally.
At start-up:
```
[INFO] *** listening on host01:43046 ***
[INFO]
[INFO] To view suite daemon contact information:
[INFO]  $ cylc get-suite-contact junk
[INFO]
[INFO] Other ways to see if the suite is still running:
[INFO]  $ cylc scan -n '\bjunk\b' host01
[INFO]  $ cylc ping -v --host=host01 junk
[INFO]  $ ssh host01 "pgrep -a -P 1 -fu $USER 'cylc-r.* \bjunk\b'" <--------------
```

The marked line is meant to print the parent PID and command line of a cylc-run or cylc-restart process for the suite, not those of the sub-process pool.

But it's been brought to my attention that some sites have a less capable version of pgrep:
```
pgrep: invalid option -- a
Usage: pgrep [-flvx] [-d DELIM] [-n|-o] [-P PPIDLIST] [-g PGRPLIST] [-s SIDLIST]
[-u EUIDLIST] [-U UIDLIST] [-G GIDLIST] [-t TERMLIST] [PATTERN]
```

### LINKED ISSUE #2500: cylc__job_finish (and cylc task message) 'priority' variable name
Hi,

I'm looking at https://github.com/cylc/cylc/blob/master/lib/cylc/job.sh#L176 and noticed that 'priority' is really a `severity`. I would suggest it should be renamed to `severity` to conform better with https://en.wikipedia.org/wiki/Syslog

It looks like this terminology persists throughout cylc-message and task_message.py, and I think they should all be updated to severity to better reflect what they are.

### PR DESCRIPTION
**Pull request by matthewrmshin**
Including:
* "suite server program" instead of "suite daemon" - close #2471 
* "severity" instead of "priority" for logging and task messaging - close #2500
* simple "ps" command instead of "pgrep" to list suite server program - close #2499

### CODE REVIEW

**Comment by matthewrmshin** [comment]
Codacy's failure is nonsense.

File: bin/cylc-message [hunk 1]
```diff
@@ -39,10 +39,11 @@ def main():
     parser = OptionParser(__doc__)
 
     parser.add_option(
-        "-p", "--priority", metavar="PRIORITY", type="choice",
+        "-s", "--severity", "-p", "--priority",
+        metavar="SEVERITY", type="choice",
         choices=['NORMAL', 'WARNING', 'CRITICAL', 'CUSTOM'],
-        help="message priority: NORMAL, WARNING, or CRITICAL; default NORMAL.",
-        action="store", dest="priority", default="NORMAL")
+        help="message severity: NORMAL, WARNING, or CRITICAL; default NORMAL.",
```
Comment (hjoliver) [inline bin/cylc-message]: The help message needs to include `CUSTOM` as well.

Comment (matthewrmshin) [reply bin/cylc-message]: Done.

**Comment by hjoliver** [comment]
(evidently minor conflicts on other people's branches can be resolved via the GitHub web editor)

**Pull request merged**